pennylane/qchem/__init__.py

```diff
@@ -54,7 +55,7 @@
 )
 from .molecule import Molecule
 from .observable_hf import fermionic_observable, qubit_observable, jordan_wigner, simplify
-from .particle_number import particle_number
+from .number import particle_number

```

Comment:
The duplication in the name of the module and its function confuses sphinx. The module name is not so important as it does not face the user.